src/main/java/jewellery/inventory/service/ResourceInUserService.java

```diff
@@ -28,12 +28,17 @@
 import jewellery.inventory.repository.ResourceRepository;
 import jewellery.inventory.repository.UserRepository;
 import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @RequiredArgsConstructor
 public class ResourceInUserService implements EntityFetcher {
+  private static final Logger logger = LogManager.getLogger(ResourceInUserService.class);
+  private static final String RESOURCE_ID = "}, Resource ID: {";
+  private static final String QUANTITY = "}, Quantity: {";
```

Comment:
Same with these. Let's just have the strings hardcoded in the message. I understand the desire to try and reuse parts of the messages but if we do it, we should go with a more complex approach.